Build failed

I was trying to package rgbds (latest release version 0.3.8) for Void Linux and during the compilation step with make this happened:
```
src/asm/asmy.c:480:23: error: conflicting types for 'yylex'
 # define YYLEX_DECL() yylex(void)
                       ^~~~~
src/asm/asmy.c:485:5: note: in expansion of macro 'YYLEX_DECL'
 int YYLEX_DECL();
     ^~~~~~~~~~
In file included from include/asm/fstack.h:20,
                 from src/asm/asmy.y:21:
include/asm/lexer.h:66:10: note: previous declaration of 'yylex' was here
 uint32_t yylex(void);
          ^~~~~
```

Gonna list the dependencies used:
- pkg-config-0.29.2_2
- byacc-20180609_1
- flex-2.6.4_3
- libpng-devel-1.6.36_1

--- a/include/asm/lexer.h
+++ b/include/asm/lexer.h
@@ -63,7 +63,7 @@ void lex_Init(void);
 void lex_AddStrings(const struct sLexInitString *lex);
 void lex_SetBuffer(char *buffer, uint32_t len);
 int yywrap(void);
-uint32_t yylex(void);
+int yylex(void);
 void yyunput(char c);
 void yyunputstr(char *s);
 void yyskipbytes(uint32_t count);

--- a/src/asm/lexer.c
+++ b/src/asm/lexer.c
@@ -879,7 +879,7 @@ static uint32_t yylex_MACROARGS(void)
 	fatalerror("Internal error in %s", __func__);
 }
 
-uint32_t yylex(void)
+int yylex(void)
 {
 	switch (lexerstate) {
 	case LEX_STATE_NORMAL:

